src/main/resources/config.yml

```diff
@@ -13,6 +13,7 @@ options:
   legacy-ore-washer: false
   legacy-dust-washer: false
   legacy-ore-grinder: true
+  legacy-gold-pan: false
```

Comment:
I hate this "legacy" naming options and I do not think this should be in the config. It's specific to the gold pan so it should be in items.yml

Please give this a better name and make it an ItemSetting

Reply:
Can you please provide a better name that would be acceptable? I do not wish to waste time creating a different but equally unacceptable alternative. Unless the legacy options for the ore washer, electric dust washer, and ore grinder are updated as well, Slimefun will end up with different methods to accomplish basically the same thing for different machines.

Reply:
Just think of an actually descriptive name. Like what the hell is "legacy" supposed to do? Without being told, you don't know.

The existing options have been there for a long time (literally years). All other options are in items.yml, this is the standard. If you want to revert behaviour for other items you check items.yml.

Reply:
> The existing options have been there for a long time (literally years).

Yeah, I know. I created them. So it's my fault they exist (well, not legacy-ore-grinder, not sure who added that one) and if it's not right I'll do what I can to correct it. I noticed there is an option that starts with override-, so maybe something along those lines since we want to override the current behavior of the machine and make it behave like it did in an older version of Slimefun (the "older version" part is what made me choose the "legacy-*" option names in the first place.) But the reason we need the old behavior is because without cargo networks the current behavior makes the machine practically useless, so perhaps it should have a name related to "cargo-net-disabled"? Do either of these cause a good descriptive option name to pop into your head?

Reply:
Converted to an ItemSetting named "override-output-limit". But did I break this PR with my rebase? Let me know if I should close this PR and create a fresh one.